### PR TITLE
fix: interpolation of default branches comes with quotes

### PR DESCRIPTION
**Pull request by matthugs**
so extra quotes here is unnecessary & in fact causes bugs!

see also https://github.com/openedx/.github/pull/21

### CODE REVIEW

File: workflow-templates/upgrade-python-requirements.yml [hunk 1]
```diff
@@ -8,13 +8,13 @@ on:
       branch:
         description: "Target branch against which to create requirements PR"
         required: true
```
Comment (sarina) [inline workflow-templates/upgrade-python-requirements.yml:10]:
```suggestion
        required: true
        # If copying this template manually, you must provide your default branch name
        # in quotes, such as 'master'
```

File: workflow-templates/upgrade-python-requirements.yml [hunk 2]
```diff
@@ -8,13 +8,13 @@ on:
...
-        default: '$default-branch'
+        default: $default-branch
 
 jobs:
   call-upgrade-python-requirements-workflow:
     uses: edx/.github/.github/workflows/upgrade-python-requirements.yml@master
     with:
```
Comment (sarina) [inline workflow-templates/upgrade-python-requirements.yml:16]:
```suggestion
    with:
    # If copying manually, also provide your default branch name in quotes here
```

**Review by sarina — APPROVED** [review]
Suggesting a change to people who may be referencing this template outside of the UI

**Comment by matthugs** [comment]
Screenshot 2022-09-19 at 12 14 23 PM
confirmed this quotes the default branch only once by trying here: https://github.com/edx/portal-designer/new/master?filename=.github%2Fworkflows%2Fupgrade-python-requirements.yml&workflow_template=upgrade-python-requirements

**Pull request merged**